src/ios/orb.yml

```diff
@@ -63,32 +85,35 @@ jobs:
                         -scheme "<< parameters.scheme >>" \
                         -configuration "<< parameters.configuration >>" \
                         -destination "platform=iOS Simulator,id=$SIMULATOR_UDID" \
-                        build-for-testing | bundle exec xcpretty
```

Comment:
`xcpretty` is bundled with the CircleCI images and I think using the global version of it doesn’t pose much of a risk since it is only used for CI builds anyway.

The change means it’s not required for every repo to have a Gemfile with `xcpretty` in it.

Reply:
Sounds good!